core/lib/types/src/blob.rs

```diff
@@ -0,0 +1,21 @@
+use zksync_basic_types::protocol_version::ProtocolVersionId;
+
+/// Returns the number of blobs supported by each VM version
+pub fn num_blobs_required(protocol_version: &ProtocolVersionId) -> usize {
+    if protocol_version.is_pre_1_5_0() {
+        2
+    } else {
+        16
+    }
+}
+
+/// Returns the number of blobs created within each VM version
+pub fn num_blobs_created(protocol_version: &ProtocolVersionId) -> usize {
```

Comment:
I think we can only have `num_blobs_created`. I also would rename method to `num_blobs_supported` or `max_num_blobs` and add a comment that this number represents max number of blobs supported by protocol for given protocol version. Add mention that this number also specifies number of system logs with blob linear hashes

Reply:
we have different levels of blobs that are supported by the circuits (reflected in the batch commitment) and in the protocol

Reply:
but if I'm not mistaken numbers are different only for 1.4.1 and it's explicitly covered with
```
if common.protocol_version.is_1_4_1() {
    // We are using zeroes as commitments to the KZG pubdata as per convention.
    result.extend(H256::zero().as_bytes());
    result.extend(H256::zero().as_bytes());
    result.extend(H256::zero().as_bytes());
    result.extend(H256::zero().as_bytes());
}
```

Reply:
Oh, sorry, I see, it's 6 vs 16 for 1.5.0. Then yeah, let's keep two functions, let's also remove the if in my comment above. And also, `num_blobs_required` should return 0 for pre-1.4.1 versions